persistence-jdbc/core/src/main/resources/play/reference-overrides.conf

```diff
@@ -2,21 +2,33 @@ jdbc-journal.slick {
   profile = ${?jdbc-defaults.slick.profile}
   # for backward compatibility only, will be removed in 1.5.0 (@deprecated)
   driver = ${?jdbc-defaults.slick.driver}
-  jndiName = ${jdbc-defaults.slick.jndiName}
+
+  # need to unset jndiName, we must force the akka-persistence-jdbc plugin
+  # to always use the pre-configured DB with AsyncExecutor
+  jndiName = ""
```

Comment:
All configs that will be read by akka-persistence-jdbc should not have a jndiName configured.